### PR TITLE
Note that symfony/process is a requirement for running tests

### PR DESCRIPTION
**Pull request by ddeboer**

### CODE REVIEW

File: doc/testing-your-application.rst [hunk 1]
```diff
@@ -57,7 +57,13 @@ Configuration
 '''''''''''''
 
 By default, the ``VarnishTestCase`` starts and stops a Varnish server for you.
-All you have to do is to set your Varnish configuration (VCL) file.
+Make sure ``symfony/process`` is available in your project:
+
+.. code-block:: bash
+
+    $ composer require symfony/process
```
Comment (dbu) [inline doc/testing-your-application.rst]: the same is true for nginx. can you update that too?

**Comment by dbu** [comment]
oh, this is completely undocumented for the symfony test case. i will add documentation.

File: doc/testing-your-application.rst (hunk 1 above)
Comment (ddeboer) [inline doc/testing-your-application.rst]: Done.

**Comment by ddeboer** [comment]
Do you want to do that here, or separately? I guess when that’s done we can release 1.4.0.

**Comment by dbu** [comment]
i will do it in a separate PR

**Pull request merged**